_data/interlang.yml

```diff
@@ -1,27 +1,27 @@
 # LANGUAGE_CODE:
-#  bg: 
-#  bn: 
-#  cn: 
-#  de: 
-#  en: 
-#  es: 
-#  fr: 
-#  gr: 
-#  id: 
-#  it: 
-#  jp: 
-#  ko: 
-#  ms: 
-#  "no": 
-#  pl: 
-#  pt: 
-#  ru: 
-#  sk: 
-#  ta: 
-#  th: 
-#  tr: 
-#  uk: 
-#  vi: 
+#  bg:
+#  bn:
+#  cn:
+#  de:
+#  en:
+#  es:
+#  fr:
+#  gr:
+#  id:
+#  it:
+#  ja:
+#  ko:
+#  ms:
+#  "no":
+#  pl:
+#  pt:
+#  ru:
+#  sk:
+#  ta:
+#  th:
+#  tr:
+#  uk:
+#  vi:
```

Comment:
the trailing space from all these lines should not be removed.

Reply:
other than this, this PR looks good to me

Reply:
🙆‍♂️ f4371af